instrumentation/opentelemetry-instrumentation-asgi/tests/test_asgi_middleware.py

```diff
@@ -801,18 +801,15 @@ class TestAsgiApplicationRaisingError(AsgiTestBase):
     def tearDown(self):
         pass
 
-    @mock.patch(
-        "opentelemetry.instrumentation.asgi.collect_custom_request_headers_attributes",
-        side_effect=ValueError("whatever"),
-    )
-    def test_asgi_issue_1883(
-        self, mock_collect_custom_request_headers_attributes
-    ):
```

Comment:
IMO this was a really bad way to test this. The alternative I'm proposing here avoids monkey patching an unrelated function, is easier to understand and is even less LOC.